docs/howto/manage-charm-design.md

```diff
@@ -0,0 +1,65 @@
+(how-to-manage-charm-design)=
```

Comment:
```suggestion
(manage-charm-design)=
```

This is the convention I've followed everywhere else (now "how-to" in the anchors).